com.github.Inochi2D.Inochi-Creator.yml

```diff
@@ -0,0 +1,164 @@
+id: com.github.Inochi2D.Inochi-Creator
+runtime: org.kde.Platform
+runtime-version: "5.15-21.08"
+sdk: org.kde.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.ldc
+  - org.freedesktop.Sdk.Extension.dmd
+command: "inochi-creator"
+finish-args:
+    - "--device=all" #OpenGL rendering, webcams (may be required? not sure)
+    - "--socket=x11"
+    - "--share=ipc"
+    - "--socket=wayland"

```

Comment:
does it support wayland?

Reply:
It does, and it does some checks to see what it's running on: https://github.com/Inochi2D/inochi-creator/commit/62099f3c6228a34c1d0127abd51a78802d5bddde

I have X and Wayland as permissions since KDialog seems to take longer to start if one or the other is blocked.